src/include/mpir_objects.h

```diff
@@ -416,6 +416,8 @@ typedef struct MPIR_Object_alloc_t {
     int initialized;            /* */
     void **indirect;            /* Pointer to indirect object blocks */
     int indirect_size;          /* Number of allocated indirect blocks */
+    int num_allocated;          /* Number of allocated objects including both direct and indirect */
```

Comment:
By "allocated", you really mean its "total capacity". Could you add "total capacity" into the comment?

Reply:
Thanks! I will fix it.

Reply:
I updated the comment:
```c
/* Total capacity of this allocator including both direct and indirect */
```